qp_qiime2/__init__.py

```diff
@@ -167,43 +164,32 @@
                     vals = list(predicate.iter_boundaries())
                     data_type = 'choice:%s' % dumps(vals)
                     default = vals[0]
+
+                qname = q2plugin.name
+                mid = m.id
+                # if we are in the diversity plugin, the method starts with
+                # alpha/beta, and the parameter is 'metric', we might want
+                # to replace the technical names for user friendly ones
+                value_pair = (mid, pname)
+                if qname == 'diversity' and value_pair in RENAME_COMMANDS:
+                    # converting to list to the serialize doesn't complaint
```

Comment:
I don't understand what this comment means. May need to rephrase.

Reply:
Good point, as minimal change it should be:`# converting to list to the serialize doesn't complaint` ->  `# converting to list so the serialize doesn't complaint`; however, not sure if that actually answers your question. The idea is that these values are gonna be converted to json ("serialized") and this action has issues with sets; thus converting to list ... Now, will that small change help with this or should I add something larger; if larger, would you mind writing exactly what you think will be the most helpful. Thanks.

Reply:
Ah to -> so. No, I think that's good.